docs/standard/modern-web-apps-azure-architecture/architectural-principles.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Architectural principles | Microsoft Docs 
```

Comment:
nit: we're not adding | Microsoft Docs to docs

Reply:
fixed across all files.